guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,330 @@
+- routing 
+- complex backups
+Lightning services offered by third-parties aim to solve these issues.
+
+This section covers two categories of lightning services: Lightning service providers (LSPs), which focus on onboarding and providing connectivity to lightning users, and [lightning wallet servers]({{ '/guide/how-it-works/lightning-services/#what-are-lightning-wallet-servers' | relative_url }}), which provide additional services not relating to connectivity.
+
+## What is a Lightning service provider?
+
+Internet service providers (ISPs) help users connect to the internet. Similarly, Lightning service providers (LSPs) help users connect to the lightning network while maintaining [self-custody]({{ '/guide/getting-started/principles/#self-custody' | relative_url }}).
+
+They do this by being well connected in the network, opening channels, and offering inbound liquidity to users. They often charge a fee for their services.
+
+You can learn more about how liquidity works on our [liquidity]({{ '/guide/how-it-works/liquidity/' | relative_url }}) page.
+
+Some LSPs may also offer channel management, routing, backups, and [other services]({{ '/guide/how-it-works/lightning-services/#what-are-lightning-wallet-servers' | relative_url }}), but these are not limited to being offered by LSPs.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Inbound-liquidity-as-a-service.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Inbound-liquidity-as-a-service@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Inbound-liquidity-as-a-service-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Inbound-liquidity-as-a-service-mobile@2x.png"
+   alt-text = "Graphic depicting how LSPs offer liquidity as a service"
+   width = 800
+   height = 348
+%}
+
+## How do LSPs connect users to lightning?
+
+To use lightning, a self-custodial user needs at least one  [payment channel]({{ '/guide/getting-started/technology-primer/#what-is-a-payment-channel' | relative_url }}). To send payments, this channel needs outbound liquidity, which the user usually provides. And to receive payments, this channel needs inbound liquidity, which needs to be provided by someone else.
+
+Finding someone offering inbound liquidity and opening a channel with them can be difficult, especially for new users. LSPs provide inbound liquidity and open channels for users.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/How-is-service-offered.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/How-is-service-offered@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/How-is-service-offered-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/How-is-service-offered-mobile@2x.png"
+   alt-text = "Graphic depicting how LSPs offer inbound liquidity as a service to users"
```

Comment:
Alt-texts should include not phrases like "Graphic of..." ([tips on alt text](https://www.a11yproject.com/posts/alt-text/)).